src/storage-controller/src/lib.rs

```diff
@@ -520,6 +520,9 @@ where
         // do_migration(&mut durable_metadata)?;
         // self.upsert_collection_metadata(&mut durable_metadata, remap_shard_migration_delta)
         //     .await;
+
+        // TODO: backfill the timeline-id for postgres sources that were created
```

Comment:
The backfill will happen in a catalog migration, or you could try to do it reactively when someone uses `ALTER SOURCE`.

Reply:
I pushed some commits to take care of this but neglected to remove the TODO––will take care of that once CI finishes.

Reply:
Oh you'd already removed it--ty